packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -294,11 +294,12 @@ const Updates = decorate([
             <CardBlock>
               <fieldset disabled={COMMUNITY}>
                 <p>
-                  {_('xoaBuild')} {state.xoaBuild} - {_('currentVersion')}{' '}
+                  {_('currentVersion')}{' '}
                   {defined(
                     () => state.installedPackages['xen-orchestra'],
                     'unknown'
                   )}{' '}
+                  - {_('xoaBuild')} {state.xoaBuild}{' '}
```

Comment:
I think we could even put parentheses around or use `text-muted` to highlight that it's less important than *current version*.